interface/usergroup/user_admin.php

```diff
@@ -228,11 +228,28 @@ function authorized_clicked() {
 <body class="body_top">
 
 <div class="container">
-
+    <?php
+    /*  Get the list ACL for the user */
+    $is_super_user = acl_check('admin', 'super');
+    $acl_name=acl_get_group_titles($iter["username"]);
+    $bg_name='';
+    $bg_count=count($acl_name);
+    $selected_user_is_superuser = false;
+    for ($i=0; $i<$bg_count; $i++) {
+        if ($acl_name[$i] == "Emergency Login") {
```

Comment:
"Emergency Login" doesn't look like a reliable identifier... I think you want to find or invent a function that gets gacl_aro_groups.value instead of gacl_aro_groups.name.

Reply:
... and then compare with "breakglass" instead of "Emergency Login".

Reply:
This is old code.. I prefer do not touch..